modules/core/04-channel/types/events.go

```diff
@@ -8,11 +8,19 @@ import (
 
 // IBC channel events
 const (
-	AttributeKeyConnectionID       = "connection_id"
-	AttributeKeyPortID             = "port_id"
-	AttributeKeyChannelID          = "channel_id"
-	AttributeKeyChannelState       = "channel_state"
-	AttributeVersion               = "version"
+	AttributeKeyConnectionID   = "connection_id"
+	AttributeKeyPortID         = "port_id"
+	AttributeKeyChannelID      = "channel_id"
+	AttributeKeyChannelState   = "channel_state"
+	AttributeKeyVersion        = "version"
+	AttributeKeyConnectionHops = "connection_hops"
+	AttributeKeyOrdering       = "ordering"
+	AttributeKeyTimeout        = "timeout"
```

Comment:
I realised that for packet events we have two separate attributes, one for the height, one for the timeout. Should we split here as well or change later on the packet event to use a single attribute?

Reply:
I guess we can resolve this from discussion in slack?